src/main/java/org/prebid/server/bidder/aja/AjaBidder.java

```diff
@@ -122,16 +122,9 @@ private ExtImpAja parseExtAJA(Imp imp, List<BidderError> errors) {
     @Override
     public Result<List<BidderBid>> makeBids(HttpCall<BidRequest> httpCall, BidRequest bidRequest) {
         final int statusCode = httpCall.getResponse().getStatusCode();
-        if (statusCode != HttpResponseStatus.OK.code()) {
-            if (statusCode == HttpResponseStatus.NO_CONTENT.code()) {
-                return Result.of(Collections.emptyList(), Collections.emptyList());
-            } else if (statusCode == HttpResponseStatus.BAD_REQUEST.code()) {
-                return Result.emptyWithError(BidderError.badInput(
-                        String.format("Unexpected status code: %d", statusCode)));
-            } else {
-                return Result.emptyWithError(BidderError.badServerResponse(
-                        String.format("Unexpected status code: %d", statusCode)));
-            }
+
+        if (statusCode == HttpResponseStatus.NO_CONTENT.code()) {
+            return Result.of(Collections.emptyList(), Collections.emptyList());
```

Comment:
Have an idea here to create separate method like `Result.empty()`. This makes code  a little bit cleaner.

Reply:
done